src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1510,6 +1515,38 @@ class spell_dru_wild_growth : public SpellScriptLoader
     }
 };
 
+// 210706 - Gore
+class spell_dru_gore : public AuraScript
+{
+    PrepareAuraScript(spell_dru_gore);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_GORE_PROC });
+    }
+
+    void HandleProc(AuraEffect* aurEff, ProcEventInfo& procInfo)
+    {
+        if (SpellInfo const* spellInfo = procInfo.GetSpellInfo())
+        {
+            if (spellInfo->Id == SPELL_DRUID_BEAR_THRASH || spellInfo->Id == SPELL_DRUID_SWIPE ||
+                spellInfo->Id == SPELL_DRUID_MAUL || spellInfo->Id == SPELL_DRUID_BASE_MOONFIRE)
+            {
+                Unit* owner = GetUnitOwner();
+
+                float chance = aurEff->GetAmount();
+                if (roll_chance_f(chance))
```

Comment:
Custom chance should either be stored in spell_proc table (only if not affected by mods, this spell is so use the other method) or rolled in DoCheckProc hook